File: graffiti/src/main/debug/java/com.ue.graffiti/DebugApplication.java
package com.ue.graffiti;

import android.app.Application;

import com.ue.graffiti.util.SPUtils;

/**
 * Created by hawk on 2018/1/23.
 */

public class DebugApplication extends Application{
    @Override
    public void onCreate() {
        super.onCreate();
        SPUtils.init(this);
    }
}

File: graffiti/src/main/debug/java/com.ue.graffiti/DebugApplication.kt
package com.ue.graffiti

import android.app.Application
import android.support.v7.app.AppCompatDelegate
import com.ue.library.util.SPUtils

/**
 * Created by hawk on 2018/1/23.
 */

class DebugApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true)
        SPUtils.init(this)
    }
}
